launcher/plans/translator_nest_to_tvb_on_local.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_action_plan>
+        <title>Test the translator Nest to TVB</title>
+        <description>Test the translator Nest to TVB</description>
+        <parameters>
+            <par_000>
+                <par_name>CO_SIM_XML_ACTIONS_DIR</par_name>
+                <par_value>${CO_SIM_TVB_NEST}/launcher/actions</par_value>
+            </par_000>
+            <par_001>
+                <par_name>CO_SIM_EXECUTION_ENVIRONMENT</par_name>
+                <par_value>Local</par_value>
+            </par_001>
+        </parameters>
+        <variables>
+            <!-- The variables under this section, will be created dynamically and held on a python dictionary
+                 NOTE: the NAME of the variable is the dictionary KEY to have access to its value
+            -->
+            <var_000>
+                <var_name>CO_SIM_DELAY</var_name>
+                <var_value>100</var_value>
+            </var_000>
+            <var_001>
+                <var_name>CO_SIM_RESULTS_DIR</var_name>
+                <!-- NOTE:  CO_SIM_RUNTIME_RESULTS_DIR will be
+                            generated on run-time by the configuration manager
+                 -->
+                <var_value>CO_SIM_RUNTIME_RESULTS_DIR</var_value>
+            </var_001>
+            <var_002>
+                <var_name>CO_SIM_INPUT_FILE</var_name>
+                <var_value>input/0.txt</var_value>
+            </var_002>
+            <var_003>
+                <var_name>CO_SIM_OUTPUT_FILE</var_name>
+                <var_value>output/0.txt</var_value>
+            </var_003>
+        </variables>
+        <action_plan>
+            <!-- prolegomenon -->
+            <action_000>
+                <action_type>CO_SIM_ACTION_SCRIPT</action_type>
+                <action_script>initial_spikes_generator.xml</action_script>
+                <action_launch_method>synchronous</action_launch_method>
+            </action_000>
+
+            <action_002>
+                <!-- waiting until the preceding action finishes -->
+                <action_type>CO_SIM_EVENT</action_type>
+                <action_event>CO_SIM_LAUNCHER_WAIT_FOR_SYNCHRONOUS_ACTIONS</action_event>
+            </action_002>
+
+            <!-- main plan action proceeding -->
+            <action_004>
+                <action>transformer_nest_to_tvb.xml</action>
+                <launching_method>asynchronous</launching_method>
```

Comment:
It seems like inconsistent tag names: for example <action_launch_method> and <launching_method>. It might be better to standardize the XML tag names/ XML schema.